vortex-array/src/compute/compare.rs

```diff
@@ -159,6 +160,24 @@ pub fn compare(
     Ok(result)
 }
 
+/// Helper function to compare empty values with arrays that have external value length information
+/// like `VarBin`.
+pub fn compare_to_empty<P, I>(lengths: I, op: Operator) -> BooleanBuffer
```

Comment:
I'm not very happy with putting this function here, but it seems preferable to having it in multiple encodings

Reply:
We share logic between varbin and varbinview, by putting the logic in varbin. Maybe we can do that same?

Reply:
This seems fine to me. We should leverage it in a bunch of places: fsst, varbin, varbinview, and list.

Reply:
the problem here is that it has to have to level `pub` because `fsst` is not a canonical encoding and lives in its own crate.